UPGRADE.md

```diff
@@ -8,6 +8,16 @@ awareness about deprecated code.
 
 # Upgrade to 4.0
 
+## Removed static keyword from `Comparator::compareSchemas($fromSchema, $toSchema)` signature
```

Comment:
Since there's no method overloading in PHP, there's no need to specify the full method signature. `Comparator::compareSchemas()` should be sufficient and consistent with the rest of the document.